.travis.yml

```diff
@@ -4,13 +4,18 @@ notifications:
 branches:
     only:
         - master
+        - format-method
```

Comment:
I'm reading your comments in the PR thread and seeing that message about CI failures. Once this is in a state ready to merge we can remove this extra branch, right?

Reply:
Yes, I'll squash the commits, rebase out the development branch, and provide some examples.